pump-jack-motion-monitoring/README.md

```diff
@@ -0,0 +1,132 @@
+
+```json
+{"req":"card.motion.mode","start":true,"sensitivity": -1,"seconds":60}
+```
+
+Since we're not using a host, we need to configure the Notecard to automatically capture motion data with [`card.motion.track`](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-track). `"start":true` starts automated motion tracking, which will send a motion Note in the default `_motion.qo` Notefile at least every 20 minutes (`"minutes":20`) and by examining the last 20 buckets (`"count":20`) of 60 seconds each (as set in `card.motion.mode` above). The `threshold` key sets the number of buckets that need to have at least one movement in order for a tracking note to be added.
+
+```json
+{"req":"card.motion.track","start":true,"minutes":20,"count":20,"threshold":1}
+```
+
+The `card.motion.sync` request is used to automatically sync motion-triggered Notefiles on an interval specified with the `minutes` key. Here, we're setting the sync interval to once an hour and using `"start":true` to kick off automated syncs.
+
+```json
+{"req":"card.motion.sync","start":true,"minutes":60}
+```
+
+Since this application will be deployed on oil fields, we also want to use GPS/GNSS to send the location of the pump jack. But the pump jack is not a mobile asset, so we can use `seconds` with a high value of `86400` to send a heartbeat location once a day.
+
+```json
+{"req":"card.location.mode","mode":"periodic","seconds":86400}
+```
+
+Finally, another `env.default` request, this time to set the Notehub-reserved `_log` name. Setting this variable will log network registration failures so that we can debug whether we have deployed devices in an area with spotty connectivity.
+
+```json
+{"req":"env.default","name":"_log","text":"on"}
+```
+
+### Viewing the App in Action
+
+Once your Notecard has been configured, powered, deployed, and shows up online, you can view events in Notehub to confirm that everything has been set-up correctly. Using the script above, you can expect to see something like the following as events from the `_motion.qo` notefile.
+
+```json
+{"motion":22,"movements":"11210221311101111110","orientation":"face-up","temperature":20.75,"voltage":5.0401675203170188}
+```
+
+`motion` corresponds to the total number of motion events captured in the configured period of 20 minutes.
+
+`movements` is a string of base-36 characters, where each character represents the number of movements in each bucket during the sample duration. Each character will be a digit 0-9, A-Z to indicate a count of 10-35, or * to indicate a count greater than 35. Since we configured the bucket duration to be 60 seconds above, the example below shows the number of movements detected during each 60 second period.
+
+Finally, `orientation` is the position of the Notecard at the time this Note was captured.
```

Comment:
Since you also set up the project for GPS/GNSS tracking, you might want to add something along the lines of:

Additionally, because you set up your project for GPS/GNSS tracking, every 24 hours you’ll see a `_track.qo` Note that contains the pump’s location data.

```json
{"lat":12,"lon":23,"or":"whatever"}
```